app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,14 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):

```

Comment:
Your `Dwarf` class should also inherit from `ABC` because it does not implement all abstract methods from the `Player` class.